1.14.5/bullseye/Dockerfile

```diff
@@ -77,7 +77,7 @@ COPY --from=verify /verify/dogecoin.tar.gz ./
 
 # Move downloaded binaries and man pages in the container system.
 # Setuid on binaries with $USER rights, to limit root usage.
-RUN tar -xvf dogecoin.tar.gz --strip-components=1 \
+RUN tar -xvf dogecoin.tar.gz --exclude="*-qt*" --strip-components=1 \
```

Comment:
Recommend to not do this, and instead explicitly copy the files we need, as that is less error-prone. On line 82, can do `cp bin/dogecoind bin/dogecoin-cli bin/dogecoin-tx /usr/local/bin/` instead of wildcard? Same on line 81...

Reply:
done